subsquid/schema.graphql

```diff
@@ -96,10 +93,100 @@ type VestingSchedule @entity {
   from: String!
   "chain event ID"
   eventId: String!
-  "{accoundId}-{assetId}"
-  scheduleId: String!
+  "Vesting schedule ID from chain"
+  scheduleId: BigInt!
   "'To' account for the vesting schedule"
   to: String!
+  "Asset ID"
+  asset: BigInt!
   "Vesting schedule"
   schedule: Schedule!
+  "Initial locked amount"
+  totalAmount: BigInt!
+  "True if the schedule has been fully claimed"
+  fullyClaimed: Boolean!
+}
+
+enum PicassoTransactionType {
+  BALANCES_TRANSFER,
+  BALANCES_DEPOSIT,
+  BALANCES_WITHDRAW,
+  STAKING_REWARDS_REWARD_POOL_CREATED,
+  STAKING_REWARDS_STAKED,
+  STAKING_REWARDS_STAKE_AMOUNT_EXTENDED,
+  STAKING_REWARDS_UNSTAKE,
+  STAKING_REWARDS_SPLIT_POSITION,
+  BONDED_FINANCE_NEW_OFFER,
+  BONDED_FINANCE_NEW_BOND,
+  BONDED_FINANCE_OFFER_CANCELLED,
+  VESTING_SCHEDULES_VESTING_SCHEDULE_ADDED,
+  VESTING_SCHEDULES_CLAIMED,
+  VESTING_SCHEDULES_VESTING_SCHEDULES_UPDATED,
+}
+
+type PicassoTransaction @entity {
```

Comment:
This does not look like the most optimal or reusable way to handle the transactions in a generic way for both Picasso and Pablo.

Reply:
Yes, that's true. I did it because there was already a `PabloTransaction` but now I've unified them in a single `Transaction` entity, which has an optional `PabloTransaction` field with the relevant information for Pablo.